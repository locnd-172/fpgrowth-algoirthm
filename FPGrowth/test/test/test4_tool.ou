84997D:3
84997D POST:3
23508:3
23494:3
23310:3
23310 23236:3
23310 23236 23241:3
23310 23236 23241 23242:3
23310 23236 23242:3
23310 23241:3
23310 23241 23242:3
23310 23242:3
23307:3
23307 22720:3
23307 22720 22961:3
23307 22961:3
23306:3
23306 22961:3
23298:3
23298 POST:3
23297:3
23297 23293:3
23296:3
23296 23293:3
23296 23293 23295:3
23296 23295:3
23295:3
23295 23293:3
23245:3
23242:3
23242 23236:3
23242 23236 23241:3
23242 23241:3
23241:3
23241 23236:3
23235:3
23191:3
23191 22961:3
22938:3
22909:3
22909 22952:3
22721:3
22721 22489:3
22668:3
22668 POST:3
22536:3
22536 22720:3
22536 22720 85178:3
22536 22720 85178 22530:3
22536 22720 22530:3
22536 85178:3
22536 85178 22530:3
22536 22530:3
22530:3
22530 22720:3
22530 22720 85178:3
22530 85178:3
22489:3
22348:3
22326:3
22326 22328:3
22090:3
22090 POST:3
21791:3
21791 22720:3
21791 22720 22492:3
21791 22492:3
20724:3
20724 POST:3
20724 POST 22378:3
20724 POST 22378 22378:1
20724 POST 22378:1
20724 22378:3
20724 22378 22378:1
20724 22378:1
15036:3
85178:4
85178 22720:3
23480:4
23480 POST:3
23480 POST 23084:3
23480 23084:3
23293:4
23293 23236:3
23236:4
23236 84077:3
23084:4
23084 POST:4
23084 POST 21731:3
23084 21731:3
22952:4
22951:4
22951 22720:3
22951 22492:3
22951 22417:3
22951 22417 22720:3
22951 22417 22720 21212:3
22951 22417 21212:3
22951 21212:3
22951 21212 22720:3
22951 21210:3
22417:4
22417 22720:3
22417 21212:4
22417 21212 22720:3
22328:4
22328 POST:3
22139:4
22139 22720:3
21977:4
21889:4
21889 22492:3
21212:4
21212 22720:3
21210:4
84077:5
84077 22961:3
22961:5
22961 22720:3
22961 22492:3
22467:5
22467 POST:5
22378:5
22378 POST:5
22378 POST 21731:4
22378 21731:4
21731:5
21731 POST:5
22779:6
22779 POST:3
22492:6
22492 22720:3
22720:7
POST:24
