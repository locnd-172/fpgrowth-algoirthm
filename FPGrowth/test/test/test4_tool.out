15036:3
20724:3
21210:4
21212:4
21731:5
21791:3
21889:4
21977:4
22090:3
22139:4
22326:3
22328:4
22348:3
22378:5
22417:4
22467:5
22489:3
22492:6
22530:3
22536:3
22668:3
22720:7
22721:3
22779:6
22909:3
22938:3
22951:4
22952:4
22961:5
23084:4
23191:3
23235:3
23236:4
23241:3
23242:3
23245:3
23293:4
23295:3
23296:3
23297:3
23298:3
23306:3
23307:3
23310:3
23480:4
23494:3
23508:3
84077:5
85178:4
POST:24
84997D:3
20724 POST:3
21731 POST:5
22090 POST:3
22328 POST:3
22378 POST:5
22467 POST:5
22668 POST:3
22779 POST:3
23084 POST:4
23298 POST:3
23480 POST:3
20724 22378:3
21210 22951:3
21212 22417:4
21212 22720:3
21212 22951:3
21731 22378:4
21731 23084:3
21791 22492:3
21791 22720:3
21889 22492:3
22139 22720:3
22326 22328:3
22417 22720:3
22417 22951:3
22489 22721:3
22492 22720:3
22492 22951:3
22492 22961:3
22530 22536:3
22530 22720:3
22530 85178:3
22536 22720:3
22536 85178:3
22720 22951:3
22720 22961:3
22720 23307:3
22720 85178:3
22909 22952:3
22961 23191:3
22961 23306:3
22961 23307:3
22961 84077:3
23084 23480:3
23236 23241:3
23236 23242:3
23236 23293:3
23236 23310:3
23236 84077:3
23241 23242:3
23241 23310:3
23242 23310:3
23293 23295:3
23293 23296:3
23293 23297:3
23295 23296:3
84997D POST:3
20724 22378 POST:3
21731 22378 POST:4
21731 23084 POST:3
23084 23480 POST:3
21212 22417 22720:3
21212 22417 22951:3
21212 22720 22951:3
21791 22492 22720:3
22417 22720 22951:3
22530 22536 22720:3
22530 22536 85178:3
22530 22720 85178:3
22536 22720 85178:3
22720 22961 23307:3
23236 23241 23242:3
23236 23241 23310:3
23236 23242 23310:3
23241 23242 23310:3
23293 23295 23296:3
21212 22417 22720 22951:3
22530 22536 22720 85178:3
23236 23241 23242 23310:3
